apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/collectionAssertions.kt

```diff
@@ -0,0 +1,40 @@
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+
+object Empty
```

Comment:
please copy the parameterObjects from api-cc-infix-en_GB-common to the new api and remove this object

Reply:
I copied the keywords file, but it has a lot of imports from `ch.tutteli.atrium.api.cc.infix.en_GB` like:

```
import ch.tutteli.atrium.api.cc.infix.en_GB.and
import ch.tutteli.atrium.api.cc.infix.en_GB.grouped
import ch.tutteli.atrium.api.cc.infix.en_GB.ignoring
import ch.tutteli.atrium.api.cc.infix.en_GB.inAny
import ch.tutteli.atrium.api.cc.infix.en_GB.to
import ch.tutteli.atrium.api.cc.infix.en_GB.within
```
where 'infix' is unresolved reference.

These are from multiple files and I don't know if you want hem in the new infix api's common package

Reply:
Ah... my fault, not parameterObjects but keywords/keywords 
We will use parameterObjects eventually as well but just copy keywords for now

Reply:
Sorry my modifications to keywords.kt hasn't been pushed, I am gonna fix it in the next